component-samples/components-oauth/main.js

```diff
@@ -0,0 +1,43 @@
+import "./styles.css";
+
+// Import the ArcGIS Map Components
+import "@arcgis/map-components/dist/components/arcgis-map";
+import "@arcgis/map-components/dist/components/arcgis-legend";
+import "@arcgis/map-components/dist/components/arcgis-search";
+
+// Import ArcGIS core modules for OAuth authentication
+import esriConfig from "@arcgis/core/config";
+import IdentityManager from "@arcgis/core/identity/IdentityManager";
```

Comment:
We call it `esriId` at https://developers.arcgis.com/javascript/latest/api-reference/esri-identity-IdentityManager.html